openlibrary/i18n/pt/messages.po

```diff
@@ -650,7 +650,7 @@ msgstr "Emprestar"
 #: widget.html:45
 #, python-format
 msgid "Join waitlist for &quot;%(title)s&quot;"
-msgstr "Entrar na lista de espera para «%(title)s»"
+msgstr "Entrar na lista de espera para &quot;%(title)s&quot;"
```

Comment:
We can't use html entities like `&quot;` everywhere, so using the direct symbol is better.

Reply:
Ah sorry missed it was that way in the `msgid`! So it will work!